Project.toml

```diff
@@ -10,7 +10,7 @@ SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 
 [compat]
 ExpectationStubs = "0.2"
-HTTP = "0.8.5"
+HTTP = "~0.8.5"
```

Comment:
This does not change anything.
See [Pkg docs](https://julialang.github.io/Pkg.jl/v1/compatibility/index.html#Caret-specifiers-1)

`0.8.5` is the equiv to `^0.8.5` (caret specifiers are the default)
and so `0.8.5` means `0.8.5 <= version < 0.9.0`

Where as  `~0.8.5` means `0.8.5 <= version < 0.9.0`